string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,10 @@
+/* exported capitalize */
+function capitalize(word) {
+  let str = '';
+  for (let i = 0; i < word.length; i++) {
+    const firstChar = word[0].toUpperCase();
+    const rest = word.slice(1).toLowerCase();
+    str = firstChar + rest;
+  }
+  return str;
+}
```

Comment:
No loop needed! We only need to capitalize the first character, and then lowercase the rest of them, so we can utilize the `slice` method here.

```
function capitalize(word) {
  return word[0].toUpperCase() + word.slice(1).toLowerCase();
}
```